ksqldb-functional-tests/src/test/resources/query-validation-tests/cast.json

```diff
@@ -382,6 +382,21 @@
       "outputs": [
         {"topic":  "OUT", "value": {"VAL": 1}}
       ]
+    },
+    {
+      "name": "string to timestamp",
```

Comment:
@aliehsaeedii the tests look good, since you add a QTT, you'll probably need to run the `PlannedTestGeneratorTest.manuallyGeneratePlans` to create the auxiliary files which are checked into source control.

That will need to happen before the build will be green!